spec/spec_helper.rb

```diff
@@ -1,3 +1,4 @@
+#spec/spec_helper.rb
```

Comment:
I'm not quite sure what the purpose of this line is. Is this supposed to turn into a require statement later?